pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -392,6 +392,11 @@ func (c *Operation) handleActionEvents(e service.DIDCommAction) error {
 	case didexchange.Event:
 		props := v
 
+		if e.Message.Header.Type == didexchange.InvitationMsgType {
```

Comment:
What do you want me to put in TODO: I am not sure what action item is here for rest API?

Reply:
With this change, there would be mismatch between SDK client and RestAPI client. In case of SDK client, the consumer would need to approve invitation and exchange request. But at the moment (due to this internally calling continue for invitation action item), RestAPI would need approval only for exchange request. We can add this feature as part of https://github.com/hyperledger/aries-framework-go/issues/692 to match the functionality added in this PR to RestAPI.